Frontend/CO.CDP.OrganisationApp/Pages/Forms/DynamicFormsPage.cshtml.cs

```diff
@@ -164,6 +181,44 @@ private bool ValidateTextAnswer()
         return true;
     }
 
+    public bool ValidateFileUpload()
+    {           
+        var allowedFileSizeMB = 10;
+        var allowedExtensions = new List<string> { ".pdf", ".docx", ".csv", ".jpg", ".bmp", ".png", ".tif" };
+
+        if (Request?.Form == null || Request.Form.Files == null || Request.Form.Files.Count == 0)
+        {
+            ModelState.AddModelError("Answer", "No file selected.");
+            return false;
+        }
+
+        var file = Request.Form.Files["UploadedFile"];
+
+        if (file == null)
+        {
+            ModelState.AddModelError("Answer", "No file selected.");
+            return false;
+        }
+
+        var maxFileLength = allowedFileSizeMB * 1024 * 1024;
+
+        if (file.Length > maxFileLength)
+        {
+            ModelState.AddModelError("Answer", "The file size must not exceed " + allowedFileSizeMB + "MB.");
```

Comment:
Maybe use string interpol  $"The file size must not exceed {allowedFileSizeMB}MB."